guides/micronaut-data-hibernate-reactive/kotlin/src/main/kotlin/example/micronaut/domain/Genre.kt

```diff
@@ -0,0 +1,16 @@
+package example.micronaut.domain
+
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.validation.constraints.NotNull
+
+@Entity
+class Genre(@Id
+            @GeneratedValue(strategy = GenerationType.IDENTITY)
+            val id: Long?,
+
+            @NotNull
```

Comment:
should this be `@NotBlank` instead of `@NotNull`?

Reply:
I agree. The @NotNull is already addressed by Kotlin language feature